.github/workflows/publish-test-result-from-fork-template.yml

```diff
@@ -0,0 +1,68 @@
+on:
+  workflow_call:
+    inputs:
+      name:
+        required: true
+        type: string
+      check_name:
+        required: true
+        type: string
+      comment_title:
+        required: true
+        type: string
+
+jobs:
+  test-results:
+    name: Test Results
+    runs-on: ubuntu-latest
+    if: >
```

Comment:
the if condition should be outside of this workflow definition, instead give the condition from each step calling